api-reference/10 UI Components/dxFileManager/1 Configuration/onDirectoryCreating.md

```diff
@@ -5,34 +5,171 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before a directory is created.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.cancel): Boolean | Promise<void>
-<!-- Description goes here -->
+Allows you to cancel the directory creation.
 
 ##### field(e.component): dxFileManager
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "component" }
 
 ##### field(e.errorCode): Number
-<!-- Description goes here -->
+#include filemanager-error-codes
 
 ##### field(e.errorText): String
-<!-- Description goes here -->
+The error message.
```

Comment:
This field is for setting, so I think better is "Allows you to specify error message"

Reply:
ok